packages/atlas/src/components/_nft/NftWidget/NftHistory.tsx

```diff
@@ -0,0 +1,178 @@
+import { formatNumberShort } from '@/utils/number'
+
+import {
+  CopyContainer,
+  DollarValue,
+  FadingBlock,
+  HistoryItemContainer,
+  HistoryPanel,
+  JoyPlusIcon,
+  NftHistoryHeader,
+  StyledChevronButton,
+  TextContainer,
+  ValueContainer,
+} from './NftHistory.styles'
+import { Size } from './NftWidget'
+import { OwnerHandle } from './NftWidget.styles'
+
+export const NftHistory: React.FC<{ size: Size; width: number }> = ({ size, width }) => {
+  const [isOpen, toggleIsOpen] = useToggle(true)
+  const { convertToUSD } = useTokenPrice()
+
+  const dummyData: HistoryItemProps[] = [
+    {
+      date: new Date(),
+      memberAvatarUri: 'https://picsum.photos/40/40',
+      size,
+      joyAmount: 12334,
+      dollarValue: convertToUSD(12334),
+      copy: 'Bid made by',
+      memberHandle: 'Madness',
+    },
+    {
+      date: new Date(),
+      memberAvatarUri: 'https://picsum.photos/40/40',
+      size,
+      joyAmount: 12334,
+      dollarValue: convertToUSD(12334),
+      copy: 'Listed by',
+      memberHandle: 'Madness',
+    },
+    {
+      date: new Date(),
+      memberAvatarUri: 'https://picsum.photos/40/40',
+      size,
+      copy: 'Auction settled by',
+      memberHandle: 'Madness',
+    },
+    {
+      date: new Date(),
+      memberAvatarUri: 'https://picsum.photos/40/40',
+      size,
+      joyAmount: 12334,
+      dollarValue: convertToUSD(12334),
+      copy: 'Auction won by',
+      memberHandle: 'Madness',
+    },
+    {
+      date: new Date(),
+      memberAvatarUri: 'https://picsum.photos/40/40',
+      size,
+      joyAmount: 12334,
+      dollarValue: convertToUSD(12334),
+      copy: 'Bid made by',
+      memberHandle: 'Madness',
+    },
+    {
+      date: new Date(),
+      memberAvatarUri: 'https://picsum.photos/40/40',
+      size,
+      joyAmount: 12334,
+      dollarValue: convertToUSD(12334),
+      copy: 'Listed by',
+      memberHandle: 'Madness',
+    },
+    {
+      date: new Date(),
+      memberAvatarUri: 'https://picsum.photos/40/40',
+      size,
+      copy: 'Auction settled by',
+      memberHandle: 'Madness',
+    },
+    {
+      date: new Date(),
+      memberAvatarUri: 'https://picsum.photos/40/40',
+      size,
+      joyAmount: 12334,
+      dollarValue: convertToUSD(12334),
+      copy: 'Auction won by',
+      memberHandle: 'Madness',
+    },
+  ]
+
+  return (
+    <>
+      <NftHistoryHeader data-open={isOpen} data-size={size} onClick={toggleIsOpen}>
+        <Text variant={size === 'small' ? 'h300' : 'h400'}>History</Text>
+        <StyledChevronButton data-open={isOpen} iconOnly variant="tertiary" icon={<SvgActionChevronB />} />
+      </NftHistoryHeader>
+      {isOpen && (
+        <div style={{ position: 'relative' }}>
+          <FadingBlock data-size={size} width={width} />
+          <HistoryPanel width={width} data-size={size} data-open={isOpen}>
+            {/* TODO: remove dummy data */}
+            {dummyData.map((props: HistoryItemProps, index) => (
```

Comment:
I'm pretty sure there no need to declare this type, it's taken directly from `dummyData`. Correct me if I'm wrong
```suggestion
            {dummyData.map((props, index) => (
```